clients/cli/src/main/java/org/apache/gravitino/cli/commands/DeleteMetalake.java

```diff
@@ -57,6 +58,8 @@ public void handle() {
     } catch (NoSuchMetalakeException err) {
       System.err.println(ErrorMessages.UNKNOWN_METALAKE);
       return;
+    } catch (MetalakeInUseException inUseException) {
+      System.err.println(metalake + " in use, plz disable it.");
```

Comment:
Please don't use abbreviations like "plz". This error message is also a bit unclear e.g. how can a user disable it? What does disabling it mean?

Reply:
fix, thank you